qt/python/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ if ( ENABLE_WORKBENCH )
 
   # ctest targets
   set ( PYTHON_TEST_FILES
+    mantidqt/test/algorithmselector_test.py
```

Comment:
I think there is a merge-related issue here. I think these 2 lines need to be removed.